src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -433,6 +444,9 @@ private bool OnPointerEnter(PointerRoutedEventArgs args, bool isManagedBubblingE
 		{
 			// We override the isOver for the relevancy check as we will update it right after.
 			var isOverOrCaptured = ValidateAndUpdateCapture(args, isOver: true);
+			//var isHandled = PointerRoutedEventArgs.PlatformSupportsNativeBubbling
+			//	? isManagedBubblingEvent
+			//	: args.Handled;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6213625)